### ISSUE TITLE
JAWS doesn't announce change of language for content inside Legend

### ISSUE THREAD

**Issue by LauraOH** [FreedomScientific/standards-support#942]
Summary

For paragraph/span language changes that fall between the opening and closing legend element tags, designated changes of language are not announced by JAWS using the default Eloquence voice.
Expected result

I expected JAWS to announce the button label using two different accents - English for "Start /" and Spanish for "Iniciar."
Actual result

JAWS read all of the legend text in an English accent.
Example

Example code:

<fieldset><legend ><p lang="es">Disfruta de un paseo en el Ferrocarril del Zoológico de Denver. Ideal para familias con niños pequeños, el tren ofrece un breve recorrido por la pradera del zoológico, rodeada de una exuberante vegetación a la sombra de árboles centenarios.</p>
  
</legend>
[...]
</fieldset>

Additional Information
JAWS version and build number
2025, 
2026.2603.86

Operating System and version
Windows 11

Browser and version:
Chrome 148.0.7778.9

**Comment by LauraOH** [comment]
#233 
#340

**Comment by JeremiahRogersVispero** [comment]
Thanks for this post, @LauraOH . Unfortunately, the browser does not pass language changes as part of the accessible name in the accessibility tree. Other Windows screen readers we tested conveyed the same information as did JAWS.
